From bd47b3f4f5c52b6b5970d0e4e38218385f3be8d8 Mon Sep 17 00:00:00 2001
From: Keerthy <j-keerthy@ti.com>
Date: Tue, 13 Dec 2016 16:06:27 -0600
Subject: [PATCH 051/205] soc: ti: wkup_m3: Add PRCM int16 as the wake up
 source

This commit comes from:
  git://git.ti.com/processor-sdk/processor-sdk-linux.git

Add PRCM int16 as the wake up source.

Signed-off-by: Keerthy <j-keerthy@ti.com>
(cherry picked from commit 4c7e0fb7af135a545c1d46fee8fc78cd6b65c43f)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/soc/ti/wkup_m3_ipc.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/soc/ti/wkup_m3_ipc.c b/drivers/soc/ti/wkup_m3_ipc.c
index 25990ae..13d9d53 100644
--- a/drivers/soc/ti/wkup_m3_ipc.c
+++ b/drivers/soc/ti/wkup_m3_ipc.c
@@ -66,6 +66,7 @@
 static struct wkup_m3_ipc *m3_ipc_state;
 
 static const struct wkup_m3_wakeup_src wakeups[] = {
+	{.irq_nr = 16,	.src = "PRCM"},
 	{.irq_nr = 35,	.src = "USB0_PHY"},
 	{.irq_nr = 36,	.src = "USB1_PHY"},
 	{.irq_nr = 40,	.src = "I2C0"},
-- 
1.7.9.5

